build.rs

```diff
@@ -110,25 +59,27 @@ fn main() {
 
     // Write the bindings to the $OUT_DIR/bindings.rs file.
     let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
-    bindings
-        .write_to_file(out_path.join("bindings.rs"))
-        .expect("Couldn't write bindings!");
+    bindings.write_to_file(out_path.join("bindings.rs"))?;
+
+    Ok(())
 }
 
-#[cfg(all(
-    not(feature = "pkg-config"),
-    not(feature = "nobuild"),
-    feature = "bundled_proj"
-))]
-fn main() {
-    // Build PROJ from the included tar
+// returns the path of "inlude" for the built proj
+fn build_from_source() -> Result<std::path::PathBuf, Box<dyn std::error::Error>> {
+    println!("building libproj from source");
+    if let Ok(val) = &env::var("_PROJ_SYS_TEST_EXPECT_BUILD_FROM_SRC") {
+        if val != "1" {
+            panic!("for testing purposes: package was building from source but should not have been");
```

Comment:
I wanted to test some things about the build and this is the best I could come up with.

A failure looks like this: https://travis-ci.org/github/georust/proj-sys/jobs/726116275

WDYT?